03/app.js

```diff
@@ -43,16 +43,46 @@ books.getAuthor = function(isbn) {
 
 books.getTitle = function(isbn, lang) {
 
+    if(typeof this[lang] === undefined){
```

Comment:
Czy w naszym obiekcie występuje coś takiego jak `books.pl` lub `books.en`?
Raczej nie - to oznacza, że ten zapis jest niepoprawny.
W szczególności, że jeśli mamy `typeof` po lewej stronie to po prawej jest ciąg znaków czyli zabrakło cudzysłowia tj. `'undefined'`